dbml-homepage/configs/sidebars.ts

```diff
@@ -12,7 +12,29 @@ import { SidebarsConfig } from '@docusaurus/plugin-content-docs';
  Create as many sidebars as you want.
  */
 const SidebarConfigs: SidebarsConfig = {
-  docs: ['home', 'docs', 'cli', 'js-module'],
+  docs: [
+    'home',
+    'docs',
+    'cli',
+    {
+      type: 'category',
+      label: 'JS Module',
+      collapsed: true,
```

Comment:
Why don't we expand the JS Module content by default? The current sidebar has many spaces so I prefer to expand it.

Reply:
Fixed in commit https://github.com/holistics/dbml/pull/615/commits/790438dd7ef00ab26cbd219a7ffe50634ddc5923